crates/swc/Cargo.toml

```diff
@@ -29,6 +29,9 @@ concurrent = [
 debug = ["swc_ecma_visit/debug"]
 node = ["napi", "napi-derive"]
 plugin = ["swc_plugin_runner", "swc_plugin_proxy/plugin-rt"]
+swc_plugin_runner_default = ["swc_plugin_runner/default"]
```

Comment:
I'm against this change.
This will make CI much slower.
Users can depend on `wasmer` by themselves

Reply:
Users can depend on wasmer by themselves won't solve the problem

But I agree with the latter point of view. I can turn off this PR first and wait for your solution~